components/channel_header/channel_header.js

```diff
@@ -478,6 +478,12 @@ class ChannelHeader extends React.PureComponent {
                                     id='channel_header.addChannelHeader'
                                     defaultMessage='Add a channel description'
                                 />
+                                <span className='button__edit ml-1'>
+                                    <FormattedMessage
+                                        id='channel_header.editLink'
+                                        defaultMessage='(Edit)'
+                                    />
+                                </span>
```

Comment:
This will probably create a nested span tag because [we use it as default](https://github.com/mattermost/mattermost-webapp/blob/7b65d4e641b17023d30e828e09e1eead143109ba/components/intl_provider/intl_provider.jsx#L71).

so this would be a bit better:

```suggestion
                                <FormattedMessage
                                    id='channel_header.editLink'
                                    defaultMessage='(Edit)'
                                />
                                      {(message) => <span className='button__edit ml-1'>{message}</span>}
                                </FormattedMessage>
```

Reply:
Cool, thanks.